test/test_client.py

```diff
@@ -1967,6 +1967,7 @@ def test_srv_max_hosts_kwarg(self):
         "loadBalanced clients do not run SDAM",
     )
     @unittest.skipIf(sys.platform == "win32", "Windows does not support SIGSTOP")
+    @unittest.skipUnless(_is_faas(), "Non-FaaS environments raise timeouts faster")
```

Comment:
We can't skip this test like this because we don't run the test suite on faas platforms. We want this test to run locally.

Reply:
Perhaps instead we can force the metadata to be faas through monkeypatching